src/proposals/20230913/SparkGoerli_20230913.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.10;
+
+import { SparkPayloadGoerli } from '../../SparkPayloadGoerli.sol';
+
+/**
+ * @title  September 13, 2023 Spark Ethereum Proposal - Set DAI borrow spread to 0.5%, set flash loan fee to 0
+ * @author Phoenix Labs
+ * @dev    This proposal updates DAI Interest Rate Strategy borrowSpread parameter and updates FLASHLOAN_PREMIUM_TOTAL pool parameter
+ * Forum:           https://forum.makerdao.com/t/phoenix-labs-proposed-changes-for-spark-for-next-upcoming-spell/21685
```

Comment:
Forum link is incorrect, correct one here: https://forum.makerdao.com/t/upcoming-spell-proposed-changes/21801